qtop.py

```diff
@@ -11,9 +11,9 @@
 from optparse import OptionParser
 import datetime
 from collections import Counter, OrderedDict
-import os
-import re
-import yaml
+# import os
+# import re
+# import yaml
```

Comment:
why not simply drop them and rely on git commits about it? r u concerned it won't be easy to revert?